ReleaseNotes.md

```diff
@@ -1,4 +1,4 @@
 # Bluedot ReactNative Minimal Integration
 
-- Using [Bluedot plugin 2.2.0](https://github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin) (support react-native@0.69.4)
+- Using [Bluedot plugin 2.2.1](https://github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin) (support react-native@0.69.4)
 - Recreated the Minimal Integration App with react@18.0.0, react-native@0.69.4
```

Comment:
Some mentioning of the new feature Allows background location update in ReleaseNotes would be nice!